guide/popular-topics/common-questions.md

```diff
@@ -33,6 +33,33 @@ const user = <message>.mentions.users.first();
 
 </branch>
 
+### How do I unban a user?
+
+<branch version="11.x">
+
+<!-- eslint-skip -->
+
+```js
+const id = args[0];
+<guild>.unban(id);
+```
+
+</branch>
+<branch version="12.x">
+
+<!-- eslint-skip -->
+
+```js
+const id = args[0];
+<guild>.members.unban(id);
+```
+
+</branch>
+
+::: warning
```

Comment:
Warning element should be tip. There is nothing in this that requires attention, we simply point to the other section where we explain how arguments are derived from the message content.